ports/harfbuzz/vcpkg.json

```
@@ -21,6 +21,15 @@
     }
   ],
   "features": {
+    "cairo": {
+      "description": "Enable Cairo graphics library support",
+      "dependencies": [
+        {
+          "name": "cairo",
+          "default-features": false
```

```suggestion
          "default-features": false,
          "features": [
            "freetype"
          ]
```
Since `harfbuzz` requires `freetype` it would make sense to also require `freetype` feature in `cairo`. (And it's probably a required feature anyway)